changelog_unreleased/javascript/13771.md

```diff
@@ -0,0 +1,9 @@
+#### Add support for "Import Reflection" proposal (#13771 by @fisker)
+
+The Stage 2 proposal ["Import Reflection"](https://github.com/tc39/proposal-import-reflection) is now supported via Babel 7.20.0. Also keep in mind our [policy on non-standardized syntax](https://github.com/tc39/proposal-explicit-resource-management/) before using this proposed syntax feature with Prettier.
```

Comment:
Why link `policy on non-standardized syntax` to `proposal-explicit-resource-management`?😕

Reply:
Copy-pasted from old changelog.